packages/openapi-generator/src/schema-util.spec.ts

```diff
@@ -100,21 +115,22 @@ describe('collectRefs', () => {
               {
                 name: 'refProperty',
                 required: false,
-                schema: { $ref: 'ref1' }
+                schema: { $ref: 'ref1', schemaName: 'Ref1' }
               }
             ]
           },
-          { $ref: 'ref2' },
+          { $ref: 'ref2', schemaName: 'Ref2' },
```

Comment:
Please correct me. I guess `$ref` to `schemaName` should be many-to-one relation? If so, could we make a test to see the result?
I know this case is added in the `test-service.json`, but also want to see the behaviour of this function as unit test.

Reply:
Maybe I am misunderstanding, but each ref is resolved to a unique name (this is the whole point of this PR ;) ).
So no, it is a one-to-one relation.

Reply:
Let's have a call.

Reply:
done